core/editor/data/globals/endpoints/colors.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Elementor\Core\Editor\Data\Globals\Endpoints;
+
+use Elementor\Plugin;
+
+class Colors extends Base {
+	public function get_name() {
+		return 'colors';
+	}
+
+	protected function get_kit_items() {
+		$result = [];
+		$kit = Plugin::$instance->kits_manager->get_active_kit_for_frontend();
+
+		$system_items = $kit->get_settings_for_display( 'system_colors' );
+		$custom_items = $kit->get_settings_for_display( 'custom_colors' );
+
+		if ( ! $system_items ) {
+			$system_items = [];
+		}
+
+		if ( ! $custom_items ) {
+			$custom_items = [];
+		}
+
```

Comment:
There are already default empty arrays for this controls

Reply:
https://github.com/elementor/elementor/pull/11622/commits/5a93721d550eebb83830727a4a291009e80f83b4